lib/splash.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+import './checkinternet.dart';
+import './webview.dart';
+
+class Splash extends StatefulWidget {
+  final debug;
+  Splash({Key? key, required this.debug}) : super(key: key);
+
+  @override
+  State<Splash> createState() => _SplashState();
+}
+
+int checkInt = 0;
+
+class _SplashState extends State<Splash> {
+  @override
+  void initState() {
+    super.initState();
+    Future<int> a = CheckInternet().checkInternetConnection();
+    a.then((value) {
+      if (value == 0) {
+        setState(() {
+          checkInt = 0;
```

Comment:
Let's use a meaningful variable name.

Reply:
CheckInt, for check-internet-integer, not good?